<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDetails</class>
 <widget class="QDialog" name="EntryDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#EntryDetails
{
background-image: url(:/patterns/tex_grey);
background-repeat: repeat-xy;
}

QGroupBox {


 border: 1px solid rgb(144,255,23);
 border-radius: 2px; 
 margin-top: 9px;
 margin-bottom: 7px;
 padding: 0px;

background-image: url(:/patterns/woven);
background-repeat: repeat-xy;
} 

QGroupBox::title {
top:-9 ex;
left: 10px;
color: green;
subcontrol-origin: border;

padding: 2px;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;

background-image: 	url(:/patterns/carbon_dark);				
background-repeat: repeat-xy;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetHeaderBG" native="true">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/patterns/carbon_dark);
background-repeat: repeat-xy;
padding: 0px 9px 0px 9px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetHeaderCont" native="true">
        <property name="styleSheet">
         <string notr="true">background: qlineargradient(spread:pad, x1:0.178, y1:0, x2:1, y2:0, stop:0 rgba(37, 202, 0, 234), stop:1 rgba(255, 255, 255, 0));
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblName">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblDate">
           <property name="styleSheet">
            <string notr="true">background: transparent;
color: white;</string>
           </property>
           <property name="text">
            <string>Datum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutHeader">
     <property name="rightMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutInfo">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPattern">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/patterns/carbon_dark);
background-repeat: repeat-xy;
color: white;
border: 1px solid rgb(85, 85, 0);</string>
       </property>
       <property name="text">
        <string>Pattern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblLang">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/patterns/carbon_dark);
background-repeat: repeat-xy;
color: white;
border: 1px solid rgb(85, 85, 0);</string>
       </property>
       <property name="text">
        <string>Sprache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSystem">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/patterns/carbon_dark);
background-repeat: repeat-xy;
color: white;
border: 1px solid rgb(85, 85, 0);</string>
       </property>
       <property name="text">
        <string>System</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hozLayoutContent">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gBoxCode">
       <property name="title">
        <string>Code</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textBrowserCode">
          <property name="styleSheet">
           <string notr="true">background: transparent;
color: white;
border: none;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vertLayoutRightContent">
       <item>
        <widget class="QGroupBox" name="qBoxDescr">
         <property name="title">
          <string>Beschreibung</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="styleSheet">
             <string notr="true">background: transparent;
color: white;
border: none;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bttnDelete">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(144,255,23);
border-style: solid;
color: rgb(122, 182, 10);
}

QPushButton:pressed
{

background-image: 	url(:/patterns/carbon_dark);			
background-repeat: repeat-xy;

border-width: 1px;
border-color:  rgb(116, 202, 18);
border-style: outset;
color: green;
}</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblAuthor">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/patterns/carbon_dark);
background-repeat: repeat-xy;
color: white;
border: rgb(116, 202, 18);
border-style: solid;
border-width: 1px 0px 1px 0px;
padding-right: 5px;</string>
     </property>
     <property name="text">
      <string>Autor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
